src/routes/docs/tooling/command-line/installation/+page.markdoc

```diff
@@ -124,6 +124,11 @@ appwrite client --reset // Resets your CLI configuration
 appwrite client --debug // Prints your current configuration
 ```
 
+# Upgrade {% #upgrade %}
+
+To upgrade Appwrite CLI, use the same method you used during the installation process.
```

Comment:
Is this the case for all installation methods? I thought some methods may not do anything because the tool is already installed.